conf/scylla.yaml

```diff
@@ -855,3 +855,17 @@ rf_rack_valid_keyspaces: false
 # Maximum number of items in single BatchWriteItem command. Default is 100.
 # Note: DynamoDB has a hard-coded limit of 25.
 # alternator_max_items_in_batch_write: 100
+
+# io-streaming rate limiting
+# When setting this value to be non-zero scylla throttles disk throughput for
+# stream (network) activities such as backup, repair, tablet migration and more.
+# This limit is useful for user queries so the network interface does 
+# not get saturated by streaming activities.
+# The recommended value is 75% of network bandwidth
+# E.g for i4i.8xlarge (https://github.com/scylladb/scylla-machine-image/tree/next/common/aws_net_params.json):
```

Comment:
This recommendation should be the same as the one that [uses it](https://github.com/scylladb/scylla-machine-image/pull/753/files#diff-0dab45bf1000b0c810886ace949a8c9b7261d77d53cd3b16e2ad427e5fd883c5R5).
Bits to bytes and network to disk transformations should be agreed and the same.